binaries/opae.io/opae/io/utils.py

```diff
@@ -412,18 +381,18 @@ def register(self, offset, name='csr', bits=[('value', c_uint64, 64)]):
                 csr.commit()
 
 
-fpga_devices = {(0x8086, 0x09c4): "Intel PAC A10 GX",
-                (0x8086, 0x09c5): "Intel PAC A10 GX VF",
-                (0x8086, 0x0b2b): "Intel PAC D5005",
-                (0x8086, 0x0b2c): "Intel PAC D5005 VF",
-                (0x8086, 0x0b30): "Intel PAC N3000",
-                (0x8086, 0x0b31): "Intel PAC N3000 VF",
-                (0x8086, 0xaf00): "Intel N6000 ADP",
-                (0x8086, 0xaf01): "Intel N6000 ADP VF",
-                (0x8086, 0xbcce): "Intel N6000 ADP",
-                (0x8086, 0xbccf): "Intel N6000 ADP VF",
-                (0x1c2c, 0x1000): "Silicom SmartNIC N5010",
-                (0x1c2c, 0x1001): "Silicom SmartNIC N5011"}
+fpga_devices = {pci.pci_id(0x8086, 0x09c4): "Intel PAC A10 GX",
+                pci.pci_id(0x8086, 0x09c5): "Intel PAC A10 GX VF",
+                pci.pci_id(0x8086, 0x0b2b): "Intel PAC D5005",
+                pci.pci_id(0x8086, 0x0b2c): "Intel PAC D5005 VF",
+                pci.pci_id(0x8086, 0x0b30): "Intel PAC N3000",
+                pci.pci_id(0x8086, 0x0b31): "Intel PAC N3000 VF",
+                pci.pci_id(0x8086, 0xaf00): "Intel N6000 ADP",
+                pci.pci_id(0x8086, 0xaf01): "Intel N6000 ADP VF",
```

Comment:
I don't think 0xaf00/0xaf01 should be in this table.  Those id were accidentally used for a bit during IOFS-EA, but have been retired.

Reply:
will remove